test/backend.test.js

```diff
@@ -90,23 +86,25 @@ describe('Full walkthrough', function () {
     // we set this so that the metadata on subsequent messages is properly filled in
     // currently group and mailbox contracts use this to determine message sender
     vuexState.loggedIn = {
-      username: user.data().attributes.username,
-      identityContractID: user.hash()
+      username: user.decrypted().data.attributes.username,
+      identityContractID: user.contractID()
     }
   }
 
   function createSocket () {
     return createWebSocket(process.env.API_URL, { timeout: 3000, strategy: false })
   }
 
-  function createIdentity (username, email) {
+  async function createIdentity (username, email) {
     // append random id to username to prevent conflict across runs
     // when GI_PERSIST environment variable is defined
     username = `${username}-${Math.floor(Math.random() * 1000)}`
-    return sbp('gi.contracts/identity/create', {
+    const msg = await sbp('gi.contracts/identity/create', {
       // authorizations: [Events.CanModifyAuths.dummyAuth(name)],
       attributes: { username, email }
     })
+    msg.decrypted(JSON.parse)
```

Comment:
This statement is confusing, why is the return value not used?

Reply:
In order to perform the decryption on the message and store the value in `this._decrypted` for followup invocations via `msg.decrypted()` (or in your case, `msg.decryptedValue()`).